.github/workflows/check-packaging-ncc-typescript-npm.yml

```diff
@@ -45,8 +45,9 @@ jobs:
       - name: Build project
         run: |
           npm run build
+          npm run pack
```

Comment:
```suggestion
```
If we add the `ncc build` inside the `npm run build` we no longer need the pack target